### PR TITLE
chore(mme): Bazelify spgw_service_impl test

### LINKED ISSUE #12105: Bazelify lte/gateway/c/core/oai/test/spgw/test_spgw_service_impl.cpp
Bazelify `lte/gateway/c/core/oai/test/spgw_task/test_spgw_service_impl.cpp`

The commands below should pass
```bash
bazel test //lte/gateway/c/core/...
bazel test //lte/gateway/c/core/... --config=asan
bazel test //lte/gateway/c/core/... --config=lsan
```

### PR DESCRIPTION
**Pull request by jheidbrink**
Closes #12105

## Summary

Add  `cc_test` target for `test_spgw_service_impl.cpp`

## Test Plan

```
bazel test //lte/gateway/c/core/oai/test/spgw_task:spgw_service_impl_test --runs_per_test=100
bazel test //lte/gateway/c/core/oai/test/spgw_task:spgw_service_impl_test --runs_per_test=100 --config=asan
bazel test //lte/gateway/c/core/oai/test/spgw_task:spgw_service_impl_test --runs_per_test=10 --config=lsan
```

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by themarwhal — APPROVED** [review]

**Review by LKreutzer — APPROVED** [review]
In pairing.

File: lte/gateway/c/core/oai/test/spgw_task/BUILD.bazel [hunk 1]
```diff
@@ -72,3 +72,12 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_test(
+    name = "spgw_service_impl_test",
```
Comment (electronjoe) [inline lte/gateway/c/core/oai/test/spgw_task/BUILD.bazel:77]: Can we add `size = "small,"` to this test? This ensures that a timeout of 60 seconds is applied (assuming 60 seconds is sufficient, if not a larger test size can be used).  This way timeouts in CI happen without enormous blocking times.

**Review by electronjoe — APPROVED** [review]

File: lte/gateway/c/core/oai/test/spgw_task/BUILD.bazel (hunk 1 above)
Comment (jheidbrink) [reply lte/gateway/c/core/oai/test/spgw_task/BUILD.bazel:77]: Sounds good! I'll force-push that once Github lets me, currently it's giving me "Internal Server Error" :worried:

**Review by ajahl — APPROVED** [review]

**Review by nstng — APPROVED** [review]
run locally LGTM :+1: 
rebase needed

**Comment by jheidbrink** [comment]
Actually master already contains this change (via PR #12162)

**Pull request closed without merging**